docs/manuals/core/concepts/index.md

```diff
@@ -43,30 +42,33 @@ This section aims at defining the following Taipy Core concepts.
 - A [Task](task.md) (the orange boxes) can be seen as a function receiving data node(s) as input and returning
   data node(s) as output.
 - A [Job](job.md) represents a unique execution of a Task.
-- A [Pipeline](pipeline.md) represents a set of tasks connected together through data nodes, that should be executed
-  as a whole and forming a consistent algorithm.
-- A [Scenario](scenario.md) is made of one or multiple pipelines. It represents an instance of a business problem to
-  solve.
+- A [Scenario](scenario.md) represents a set of tasks connected through data nodes forming a Directed Acyclic Graph,
+  that should be executed as a whole (or as subgraphs via sequences) and create a consistent algorithm. It can
+  also contain a set of additional data nodes for related data but are not part of the executable graph.
+- A [Sequence](sequence.md) represents a set of tasks connected through data nodes, that should be executed
+  as a whole and forming a consistent algorithm. A sequence belongs to a scenario and can be thought of as a subgraph of
+  the scenario's complete graph.
 - A [Cycle](cycle.md) or work cycle is a time period corresponding to an iteration of a recurrent business problem.
   For instance, a company's sales forecast needs to be generated _every week_.
   A cycle is defined thanks to the **_Frequency_** of scenarios. For instance, if scenarios have a weekly
   frequency, then each cycle represents a particular week, and every scenario will be attached to a particular cycle
   (i.e. a week).
 - A [Scope](scope.md) represents the _visibility_ of a data node in the graph of entities, and the level of its
-  owner (Pipeline, Scenario, Cycle).
+  owner (Scenario, Cycle, Global).
 
 !!! important "Definition: Config vs Entities"
 
-    Among the concepts described in this section, **data nodes**, **tasks**, **pipelines**, and **scenarios** have
+    Among the concepts described in this section, **data nodes**, **tasks**, **sequences**, and **scenarios** have
```

Comment:
I would not talk abt sequence here. It is like Cycle. We don't need to mention it in this "important" message.

```suggestion
    Among the concepts described in this section, **data nodes**, **tasks**, and **scenarios** have
```